packages/api/src/models/fieldModel.js

```diff
@@ -44,6 +44,14 @@ class Field extends Model {
   static get relationMappings() {
     // Import models here to prevent require loops.
     return {
+      organic_history: {
+        modelClass: require('./organicHistoryModel'),
+        relation: Model.HasManyRelation,
+        join: {
+          from: 'field.location_id',
+          to: 'organic_history.location_id',
```

Comment:
need relationMappings so that upsertGraph can insert new organic_history while adding/updating location